### ISSUE TITLE
Warning about Assuming assembly reference 'System.Data.Common'

### ISSUE THREAD

**Issue by pitermarx** [cake-contrib/Cake.SqlServer#73]
If I have a build.cake file just with this
```
#addin nuget:?package=Cake.SqlServer&loaddependencies=true&version=3.0.0
```

and a tool's manifest like this
```
{
  "version": 1,
  "isRoot": true,
  "tools": {
    "cake.tool": {
      "version": "1.1.0",
      "commands": [
        "dotnet-cake"
      ]
    }
  }
}
```

and then run it ```dotnet cake```

I get this wall of text

```
warning CS1701: Assuming assembly reference 'System.Data.Common, Version=4.2.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' used by 'Microsoft.Data.SqlClient' matches identity 'System.Data.Common, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' of 'System.Data.Common', you may need to supply runtime policy
warning CS1701: Assuming assembly reference 'System.Data.Common, Version=4.2.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' used by 'Microsoft.Data.SqlClient' matches identity 'System.Data.Common, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' of 'System.Data.Common', you may need to supply runtime policy
warning CS1701: Assuming assembly reference 'System.Data.Common, Version=4.2.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' used by 'Microsoft.Data.SqlClient' matches identity 'System.Data.Common, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' of 'System.Data.Common', you may need to supply runtime policy
(3285,5): warning CS1701: Assuming assembly reference 'System.Data.Common, Version=4.2.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' used by 'Microsoft.Data.SqlClient' matches identity 'System.Data.Common, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' of 'System.Data.Common', you may need to supply runtime policy
(3292,5): warning CS1701: Assuming assembly reference 'System.Data.Common, Version=4.2.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' used by 'Microsoft.Data.SqlClient' matches identity 'System.Data.Common, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' of 'System.Data.Common', you may need to supply runtime policy
(3299,5): warning CS1701: Assuming assembly reference 'System.Data.Common, Version=4.2.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' used by 'Microsoft.Data.SqlClient' matches identity 'System.Data.Common, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' of 'System.Data.Common', you may need to supply runtime policy
(3306,5): warning CS1701: Assuming assembly reference 'System.Data.Common, Version=4.2.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' used by 'Microsoft.Data.SqlClient' matches identity 'System.Data.Common, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' of 'System.Data.Common', you may need to supply runtime policy
(3313,12): warning CS1701: Assuming assembly reference 'System.Data.Common, Version=4.2.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' used by 'Microsoft.Data.SqlClient' matches identity 'System.Data.Common, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' of 'System.Data.Common', you may need to supply runtime policy
```

Should I worry? I can't say I fully understand what's going on

**Comment by trailmax** [comment]
Smells like nuget version mismatch. @AdaskoTheBeAsT Any idea what is happening?

**Comment by AdaskoTheBeAsT** [comment]
I will check this - it seems it is somewhat connected with dotnet cake tool

**Comment by AdaskoTheBeAsT** [comment]
hmmm as I checked indeed there is a problem with version 1.1.0 of cake
version 1.2.0 and freshly released 1.3.0 are free from that issue
Can you please upgrade tool version and check one more time?
